relay-metrics/src/aggregation.rs

```diff
@@ -690,15 +690,10 @@ pub struct Bucket {
     pub timestamp: UnixTimestamp,
     /// The length of the time window in seconds.
     pub width: u64,
-    /// The name of the metric without its unit.
+    /// The MRI (metric resource identifier).
     ///
     /// See [`Metric::name`].
     pub name: String,
-    /// The unit of the metric value.
-    ///
-    /// See [`Metric::unit`].
-    #[serde(default, skip_serializing_if = "MetricUnit::is_none")]
-    pub unit: MetricUnit,
```

Comment:
We checked this through code review yesterday, though it would be good to add these points to the PR description @untitaker. 

> * new relays receiving `unit` will ignore it

That is correct. 

> * old relays that expect a unit will now log the metric with unit `None`. But as long as we deploy processing relays first, this should not be a problem.

Since we introduced MRIs initially, there has been a unit in all metrics names we are ingesting so far.